test/integration/guest_user_test.rb

```diff
@@ -102,9 +102,7 @@ def setup
     click_link_or_button('Create Account')
 
     assert current_path, root_url
-    within('#header') do
-      assert page.has_content?('Welcome, Harry')
-    end
+    assert page.has_content?('Welcome, Harry')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.